src/js/player.js

```diff
@@ -2767,6 +2776,8 @@ Player.prototype.options_ = {
  */
 Player.prototype.handleLoadedMetaData_;
 
+Player.prototype.handleTextData_;
```

Comment:
Can you add a jsdoc comment here, similar to how `loadeddata` is below, for example.
I can add it myself when I'm merging if it isn't done by then.